pkgs/servers/mail/mailman/python.nix

```diff
@@ -28,6 +28,10 @@ python3.override {
           hash = "sha256-WF3FFrnrBCphnvCjnD19Vf6BvbTfCaUsnN3g0Hvxqn0=";
         };
       });
+
+      readme-renderer = super.readme-renderer.overridePythonAttrs (_: {
+        propagatedBuildInputs = [ self.cmarkgfm ];
+      });
```

Comment:
What? Why do we only add this for mailman, and not the general readme-renderer package?

We were about to be able to eliminate the last remaining override…

Reply:
38 hours is also not a lot of time to leave for maintainer review…

Reply:
Sorry for merging this myself.

The dependency is an optional feature and not strictly required for `readme_renderer`: https://github.com/pypa/readme_renderer/blob/main/pyproject.toml#L34-L35. Since mailman does require this feature, I thought it would be fitting to add this to the overrides instead of adding the dependency to `readme_renderer` to follow upstream.

Any other suggestions?

Reply:
buildPythonPackage has a mechanism to express optional-dependencies, so I think we'd just use that.

https://nixos.org/manual/nixpkgs/unstable/#buildpythonpackage-parameters